pkg/vault/test/authorizer/TimelockAuthorizer.test.ts

```diff
@@ -375,72 +375,13 @@ describe('TimelockAuthorizer', () => {
           context('for a specific contract', () => {
             const where = WHERE_1;
 
-            context('when the sender has permission', () => {
-              beforeEach('grant permission', async () => {
-                await authorizer.addGranter(actionId, from, where, { from: root });
-              });
-
-              it('can grant permission for that action in that contract only', async () => {
```

Comment:
[Nothing to address here, just question] Could it be that these test messages were out of sync with the actual tests?

Reply:
Yes, the original tests were a bit of a mess